src/Bridge/Symfony/DependencyInjection/Compiler/ConfigureDependencyPass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Nelmio\Alice\Bridge\Symfony\DependencyInjection\Compiler;
+
+use RuntimeException;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class ConfigureDependencyPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        if (!$container->has('property_accessor')) {
+            throw new RuntimeException('NelmioAliceBundle requires framework_bundle.property_access to be enabled.');
```

Comment:
I would elevate this to a `LogicException` or `DomainException`: `RuntimeException` to me implies it's an exception that makes sense to catch and retry at runtime, whereas LogicException & Error is more akin to a compilation error and requires a manual fix (which is the case here).